prometheus-stats

```diff
@@ -159,6 +160,27 @@ pr_retries_count {acc}
 merged_failures {merged_failures}
 """
 
+    output += """
+# HELP top_slowest_tests The slowest tests in the last 7 days
+# TYPE top_slowest_tests gauge
+"""
+    slowest_tests = """
+        SELECT TestRuns.project, t1.testname, t1.seconds FROM Tests AS t1
+        JOIN TestRuns ON t1.run = TestRuns.id
+        WHERE TestRuns.time > strftime('%s', 'now', '-7 days')
+        AND t1.seconds > 0 order by seconds desc
+        LIMIT 200;
+    """
```

Comment:
How about this? I used max/group by to filter out duplicates, reformatted it a  bit and dropped the unnecessary table names:

```
    slowest_tests = """ 
        SELECT project, testname, max(seconds) FROM Tests AS t1
        JOIN TestRuns ON t1.run = TestRuns.id
        WHERE TestRuns.time > strftime('%s', 'now', '-7 days') AND seconds > 0
        GROUP BY project, testname
        ORDER BY seconds DESC
        LIMIT 200
    """
```

Reply:
Looks good! Thanks for the improvement!